admin-manual/installation-setup/installation/install-centos.rst

```diff
@@ -69,24 +69,24 @@ Installation instructions
    * Archivematica:
 
    Use these commands to install the repositories:
-
+      
```

Comment:
I think it is not so important as python code, but an empty line has been replaced by a line with a tab (or several blank spaces). Same for lines 74 and 83

Reply:
I updated to fix these so they have empty lines as before.